plugins/hashsum_download/web_client/templates/hashsum_download_fileInfo.jade

```diff
@@ -0,0 +1,6 @@
+if file.has('sha512')
```

Comment:
I agree. The one possible workaround is for the API document to be look different from the internal model document. I think that violates too many other Girder expectations though.